tests/e2e/config_secret.go

```diff
@@ -150,8 +150,9 @@ var _ = ginkgo.Describe("Config-Secret", func() {
 		7. Cleanup all objects created during the test
 	*/
 
-	ginkgo.It("[csi-config-secret-block][csi-config-secret-file] Update user credentials in vsphere config "+
-		"secret keeping password same for both test users", func() {
+	ginkgo.It("Update user credentials in vsphere config secret keeping password same "+
+		"for both test users", ginkgo.Label(TestSuiteLabelBlockConfigSecret, TestSuiteLabelFileConfigSecret,
```

Comment:
it can be ginkgo.Label("block", "file", "config-secret", "vanilla", "P0", "stable")